arches_lingo/src/arches_lingo/components/concept/ConceptImages/components/ConceptImagesViewer.vue

```diff
@@ -0,0 +1,195 @@
+<script setup lang="ts">
+import { inject, ref, onMounted } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+
+import FileListWidget from "@/arches_component_lab/widgets/FileListWidget/FileListWidget.vue";
+
+import { DANGER, SECONDARY, VIEW } from "@/arches_lingo/constants.ts";
+import { useConfirm } from "primevue/useconfirm";
+
+import type {
+    ConceptImages,
+    DigitalObjectInstance,
+} from "@/arches_lingo/types.ts";
+import { fetchLingoResource } from "@/arches_lingo/api.ts";
+import NonLocalizedStringWidget from "@/arches_component_lab/widgets/NonLocalizedStringWidget/NonLocalizedStringWidget.vue";
+
+const props = defineProps<{
+    tileData: ConceptImages | undefined;
+    componentName: string;
+    sectionTitle: string;
+    graphSlug: string;
+    nodegroupAlias: string;
+}>();
+
+const resources = ref<DigitalObjectInstance[]>();
+const { $gettext } = useGettext();
+const confirm = useConfirm();
+
+onMounted(async () => {
```

Comment:
will this also need to happen in the editor? If so the logic should be moved up a level.

Reply:
yes and no.  The editor is only retrieving one resource at a time.  We could still likely move it up to improve request frugality.

Reply:
going to punt on this until the follow up PR re: the editor